index.html

```diff
@@ -75,11 +75,13 @@ <h2 style="padding-top:0px;">Major Features</h2>
     </div>
   </div>
 
-  <h2>Disclaimer</h2>
+  <hr>
+
+  <a href="https://incubator.apache.org"><img src="{{ site.baseurl }}/resources/apache-incubator.png" alt="Apache Incubator"></a>
```

Comment:
I think we should link to the logo (https://path/to/incubator-logo.png) rather than add it to our own repo.